modules/lib/lib-schema/src/test/resources/test/UpdateDynamicContentSchemaHandlerTest.js

```diff
@@ -21,4 +21,403 @@ exports.updateInvalidContentSchemaType = function () {
+                        "default": false,
+                        "items": [
+                            {
+                                "formItemType": "Input",
+                                "name": "text-input",
+                                "label": "Name",
+                                "helpText": "Text input",
+                                "maximize": true,
+                                "inputType": "TextLine",
+                                "occurrences": {
+                                    "maximum": 1,
+                                    "minimum": 1
+                                },
+                                "default": {
+                                    "value": "something",
+                                    "type": "String"
+                                },
+                                "config": {}
+                            },
+                            {
+                                "formItemType": "ItemSet",
+                                "name": "minimum3",
+                                "label": "Minimum 3",
+                                "occurrences": {
+                                    "maximum": 0,
+                                    "minimum": 3
+                                },
+                                "items": [
+                                    {
+                                        "formItemType": "Input",
+                                        "name": "label",
+                                        "label": "Label",
+                                        "maximize": true,
+                                        "inputType": "TextLine",
+                                        "occurrences": {
+                                            "maximum": 1,
+                                            "minimum": 0
+                                        },
+                                        "config": {}
+                                    },
+                                    {
+                                        "formItemType": "Input",
+                                        "name": "value",
+                                        "label": "Value",
+                                        "maximize": true,
+                                        "inputType": "TextLine",
+                                        "occurrences": {
+                                            "maximum": 1,
+                                            "minimum": 0
+                                        },
+                                        "config": {}
+                                    }
+                                ]
+                            }
+                        ]
+                    },
+                    {
+                        "name": "option_2",
+                        "label": "Option 2",
+                        "default": false,
+                        "items": []
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9936826)
```suggestion
                        'items': []
```